shell/platform/darwin/ios/framework/Source/FlutterAppDelegateTest.mm

```diff
@@ -14,84 +14,145 @@
 FLUTTER_ASSERT_ARC
 
 @interface FlutterAppDelegateTest : XCTestCase
+@property(strong) FlutterAppDelegate* appDelegate;
+
+@property(strong) id mockMainBundle;
+@property(strong) id mockNavigationChannel;
+
+// Retain callback until the tests are done.
+// https://github.com/flutter/flutter/issues/74267
+@property(strong) id mockEngineFirstFrameCallback;
```

Comment:
I only just landed https://github.com/flutter/engine/pull/27854 but this is probably a better name than `blockNoInvoker`. Open to suggestions though!